frontend/src/layout/navigation/SideBar/SideBar.tsx

```diff
@@ -238,6 +239,17 @@ function Pages(): JSX.Element {
             )}
             <PageButton icon={<IconTools />} identifier={Scene.ToolbarLaunch} to={urls.toolbarLaunch()} />
             <PageButton icon={<IconSettings />} identifier={Scene.ProjectSettings} to={urls.projectSettings()} />
+            {user?.is_staff && (
+                <>
+                    <LemonSpacer />
+                    <PageButton
+                        title="Instance settings"
```

Comment:
Ah, honestly I didn't notice the sidebar button. This raises more questions – it's kind of weird:
- there are now two buttons that take you to this page – one in the sidebar, and one in the popover
- the sidebar button takes you to instance settings… but that's the third tab of the page, not the first, which is a surprising destination for a pattern that's meant for top-level pages
- "Settings" is just one of a few tabs – a sidebar item should cover all tabs of the page it leads too (otherwise the highlighting is confusing when you navigate to another page tab for example)

If we go the sidebar item route, I strongly propose making instance settings their own page, with status/metrics reachable from the popover like it is currently.